ccdproc/tests/test_image_collection.py

```diff
@@ -700,3 +700,22 @@ def test_ccds_generator_in_different_directory(self, triage_setup, tmpdir):
         # This generated an IOError before the issue was fixed.
         for _ in coll.ccds(ccd_kwargs={'unit': 'adu'}):
             pass
+
+    def test_glob_matching(self, triage_setup):
+        # We'll create two files with strange names to test glob
+        #   includes / excludes
+        one = fits.PrimaryHDU()
+        one.data = np.zeros((100, 100))
```

Comment:
This is possibly just a copy&paste but there's no point in creating such a "big image" here, could you change it to `(5, 5)` (or something like that)?